Lesson-8/Commands/ChangeAtribCommand.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Lesson_8.Commands.Base;
+
+namespace Lesson_8.Commands
+{
+    public class ChangeAttribCommand : FMCommands
+    {
+        private readonly IUserInterface _UserInterface;
+        private readonly FileManagerLogic _FileManager;
+
+        public override string? Description => "Изменение аттрибутов";
+
+        public ChangeAttribCommand(IUserInterface UserInterface, FileManagerLogic FileManager)
+        {
+            _UserInterface = UserInterface;
+            _FileManager = FileManager;
+        }
+
+        public override void Execute(string[] args)
+        {
+            if (args.Length != 2 || string.IsNullOrWhiteSpace(args[1]))
```

Comment:
Если в файле `.csproj` в раздел `PropertyGroup` добавить `<LangVersion>preview</LangVersion>`, то уже сейчас можно использовать возможности C#11
```suggestion
            if (args is not [_, { Length: > 0 }])
```
Мы говорим что `args` не должна состоять из двух элементов и второй элемент не должен быть строкой с длиной больше 0.